lock-api/src/main/java/com/palantir/lock/client/TimeLockClient.java

```diff
@@ -40,6 +41,8 @@
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.stream.Collectors;
 
+// TODO (jkong): This class almost shouldn't be a plain TimelockService, but the amount of effort that would be
```

Comment:
Are you actually going to do this?

Reply:
I removed the TODO and left my lamentation about the bad design as a plain comment.